packages/comms/package.json

```diff
@@ -39,17 +39,25 @@
+        "wsdl-cloud": "node ./lib-cjs/index.js --url=http://localhost:8010/WsCloud?wsdl --outDir=./src/services/wsdl",
         "wsdl-codesign": "node ./lib-cjs/index.js --url=http://localhost:8010/ws_codesign?wsdl --outDir=./src/services/wsdl",
         "wsdl-dali": "node ./lib-cjs/index.js --url=http://localhost:8010/WsDali?wsdl --outDir=./src/services/wsdl",
         "wsdl-dfu": "node ./lib-cjs/index.js --url=http://localhost:8010/WsDfu?wsdl --outDir=./src/services/wsdl",
         "wsdl-dfuxref": "node ./lib-cjs/index.js --url=http://localhost:8010/WsDFUXRef?wsdl --outDir=./src/services/wsdl",
-        "wsdl-logaccess": "node ./lib-cjs/index.js --url=http://localhost:8010/Ws_logaccess?wsdl --outDir=./src/services/wsdl",
+        "wsdl-elk": "node ./lib-cjs/index.js --url=http://localhost:8010/ws_elk?wsdl --outDir=./src/services/wsdl",
+        "wsdl-esdlconfig": "node ./lib-cjs/index.js --url=http://localhost:8010/ws_esdlconfig?wsdl --outDir=./src/services/wsdl",
+        "wsdl-fileio": "node ./lib-cjs/index.js --url=http://localhost:8010/WsFileIO?wsdl --outDir=./src/services/wsdl",
+        "wsdl-filespray": "node ./lib-cjs/index.js --url=http://localhost:8010/FileSpray?wsdl --outDir=./src/services/wsdl",
+        "wsdl-logaccess": "node ./lib-cjs/index.js --url=http://localhost:8010/ws_logaccess?wsdl --outDir=./src/services/wsdl",
         "wsdl-machine": "node ./lib-cjs/index.js --url=http://localhost:8010/ws_machine?wsdl --outDir=./src/services/wsdl",
+        "wsdl-packageprocess": "node ./lib-cjs/index.js --url=http://localhost:8010/WsPackageProcess?wsdl --outDir=./src/services/wsdl",
         "wsdl-smc": "node ./lib-cjs/index.js --url=http://localhost:8010/WsSMC?wsdl --outDir=./src/services/wsdl",
+        "wsdl-resources": "node ./lib-cjs/index.js --url=http://localhost:8010/ws_resources?wsdl --outDir=./src/services/wsdl",
         "wsdl-store": "node ./lib-cjs/index.js --url=http://localhost:8010/WsStore?wsdl --outDir=./src/services/wsdl",
         "wsdl-topology": "node ./lib-cjs/index.js --url=http://localhost:8010/WsTopology?wsdl --outDir=./src/services/wsdl",
         "wsdl-workunits": "node ./lib-cjs/index.js --url=http://localhost:8010/WsWorkunits?wsdl --outDir=./src/services/wsdl",
-        "wsdl": "npm-run-all --serial compile-util --parallel wsdl-account wsdl-codesign wsdl-dfu wsdl-dfuxref wsdl-machine wsdl-smc wsdl-store wsdl-topology wsdl-workunits"
+        "wsdl-all": "npm-run-all --serial compile-util --parallel wsdl-access wsdl-account wsdl-cloud wsdl-codesign wsdl-dali wsdl-dfu wsdl-dfuxref wsdl-elk wsdl-esdlconfig wsdl-fileio wsdl-filespray wsdl-logaccess wsdl-machine wsdl-packageprocess wsdl-smc wsdl-resources wsdl-store wsdl-topology wsdl-workunits",
+        "wsdl": "npm-run-all --serial compile-util --parallel wsdl-access wsdl-account wsdl-cloud wsdl-codesign wsdl-dali wsdl-dfu wsdl-dfuxref wsdl-elk wsdl-fileio wsdl-filespray wsdl-logaccess wsdl-packageprocess wsdl-smc wsdl-store wsdl-topology wsdl-workunits"
```

Comment:
Is there a different use for these two?

Reply:
wsdl-all contained some services which were not available on my installation...

Reply:
See:  https://github.com/hpcc-systems/Visualization/issues/3961